src/routers/asset_router.py

```diff
@@ -0,0 +1,60 @@
+from http import HTTPStatus
+from fastapi import APIRouter, Depends, HTTPException
+from sqlmodel import Session, select
+
+from authentication import KeycloakUser, get_user_or_raise, get_user_or_none
+from database.session import get_session
+from routers.helper_functions import get_asset_type_by_abbreviation
+from routers.resource_routers import router_list
+
+from typing import Type, Union
+
+
+def create(url_prefix: str = "") -> APIRouter:
+    router = APIRouter()
+
+    for path in [
+        f"{url_prefix}/v2/assets",
+        f"{url_prefix}/assets",
```

Comment:
```suggestion
        f"{url_prefix}/v2/assets/{{identifier}}",
        f"{url_prefix}/assets/{{identifier}}",
```
let's make it a path parameter instead of a query parameter to be consistent with the other endpoints